libevm/VM.cpp

```diff
@@ -1096,7 +1491,7 @@ void VM::interpretCases()
 #if EVM_HACK_DUP_64
 			*(uint64_t*)m_SPP = *(uint64_t*)(m_SP + n);
 #else
-			m_SPP[0] = m_SP[n];
+			new(m_SPP) u256(m_SP[n]);
```

Comment:
The stack slot being copied into may no longer hold a u256, so we construct a new one in the memory, rather than assign.  I added a comment.